spec/models/api/geographical_area_spec.rb

```diff
@@ -85,6 +85,29 @@
     end
   end
 
+  describe '.other_countries' do
+    let(:expected_members) do
+      %w[AT BE BG CY CZ DE DK EE ES EU FI FR GR HR HU IE IT LT LU LV MT NL PL PT RO SE SI SK]
+    end
+    let(:members) { described_class.other_countries.map(&:id) }
+
+    it 'returns non-eu results' do
+      expect(members.length).to be_positive
+    end
+
+    it 'returns none of the member countries of the EU' do
+      expect(members).not_to include(expected_members)
+    end
+
+    it 'doesn\'t return UK' do
+      expect(members).not_to include('UK')
+    end
+
+    it 'doesn\'t return XI' do
+      expect(members).not_to include('XI')
```

Comment:
I don't see that this is a feature of this filtering behaviour in the duty calculator?

Reply:
Probs can remove this test?

Reply:
Because XI would never be returned as country on the :xi service right?

Reply:
Yarp. It's not in our list of expected eu members

Reply:
So we're saying we expect everyone in the list of eu members plus the UK (`GB`) to be rejected. The backend separately excludes `XI` and is independently tested in the backend unit tests for the geographical area service.

So we want one eu member geographical area id to be missing (e.g. `RO`) and one non-eu member geographical area to to be included (e.g. `NI`).

I'm assuming we're relying on the automatically loaded fixtures so we can just pick one from that list? So I'd just pick one of the other countries from that file.

Reply:
The "don't include any EU country" is covered by the "returns none of the member countries of the EU" test, I'll add some specific non-EU country